labs/university-course.md

```diff
@@ -1,5 +1,5 @@
 # Lab Name
-Enterprise Blockchain Technologies - A University Course on Hyperledger Technologies
+Enterprise Blockchain Technologies - A University Course on Hyperledger Fabric
```

Comment:
The lab name in the proposal should reflect the markdown file and repository name. (This is the common standard across other submissions as well.)

As per https://wiki.hyperledger.org/display/labs; the proposal name should be appropriate (e.g., not too generic, confusing, or conflicting with other labs or projects, etc)

I would still suggest naming the labs: `University Course Fabric` (file: labs/university-course-fabric.md)

If the group name of the coursework is `Enterprise Blockchain Technologies` I could imaging putting that in the short description.

Reply:
That makes sense, thanks for your input! I've updated the proposal.